README.md

```diff
@@ -237,8 +237,4 @@ as parameters that will be passed to the test function.
 License
 -------
 
-Unless specifically indicated otherwise in a file, TF-PSA-Crypto files are
-provided under the [Apache-2.0](https://spdx.org/licenses/Apache-2.0.html)
-license. See the [LICENSE](LICENSE) file for the full text of this license.
-Contributors must accept that their contributions are made under the
```

Comment:
Shouldn't we retain this line `Contributors must accept...` (as we have done in Mbed TLS)?

Reply:
I think you're right. I didn't notice that.

Reply:
I removed it because we do not accept contributions currently. We could add it back when we open to contributions. That's the rationale behind that change.

Reply:
OK - please make a note of this somewhere so we don't forget